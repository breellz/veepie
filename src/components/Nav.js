import React from 'react';
import { NavLink } from 'react-router-dom';


export const Nav = () => (
  <header className="nav">
  <div className="content-container nav-link">
  <NavLink className="nav-link__item" to="/dashboard" activeClassName= "is-active"><h3>Dashboard</h3></NavLink>
  <NavLink className="nav-link__item" to ="/create" activeClassName ="is-active"><h3>Add Post</h3></NavLink><br />
  <NavLink className="nav-link__item" to ="/profile" activeClassName ="is-active"><h3>Profile</h3></NavLink>
  </div>
  </header>
);


export default Nav;
