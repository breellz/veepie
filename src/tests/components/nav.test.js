import React from 'react';
import { shallow } from 'enzyme'
import Nav from '../../components/Nav'

test('should render navbar correctly', ()=>{
    const wrapper = shallow(<Nav />)
    expect(wrapper).toMatchSnapshot()
})