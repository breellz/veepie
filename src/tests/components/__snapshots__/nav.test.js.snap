// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render navbar correctly 1`] = `
<header
  className="nav"
>
  <div
    className="content-container nav-link"
  >
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      className="nav-link__item"
      to="/dashboard"
    >
      <h3>
        Dashboard
      </h3>
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      className="nav-link__item"
      to="/create"
    >
      <h3>
        Add Post
      </h3>
    </NavLink>
    <br />
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      className="nav-link__item"
      to="/profile"
    >
      <h3>
        Profile
      </h3>
    </NavLink>
  </div>
</header>
`;
